modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPInstanceLocalServiceImpl.java

```diff
@@ -902,17 +902,18 @@ public void checkCPInstancesByDisplayDate(long cpDefinitionId)
 			serviceContext.setWorkflowAction(WorkflowConstants.ACTION_PUBLISH);
 
 			CPDefinition cpDefinition =
-				cpDefinitionLocalService.getCPDefinition(cpDefinitionId);
+				cpDefinitionLocalService.getCPDefinition(
```

Comment:
why here we take the cpDefinitionId form the instance? is it not suppose to be the same as cpDefinitionId?

Reply:
I believe here @brianikim made this change because the cpDefinitionId passed in is 0, while we need to check against actual products

Please confirm @brianikim